src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+            </p>
+            <CardGrid>
+              <Card
+                title="What is a wallet?"
+                description="How you manage your ETH and your Ethereum accounts."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.wallet)}
+                      alt="Illustration of a robot."
+                    />
+                  </CardImage>
+                  <ButtonLink to="/wallets/">What is a wallet?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Find a wallet"
+                description="We will help you find the right wallet based on the features you care about."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.futureTransparent)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/wallets/find-wallet/">
+                    List of wallets
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Crypto security 101"
+                description="Learn how to identify scams and how to avoid the most common tricks."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.dogeComputer)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/security/">Crypto security 101</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <Banner>
+              <BannerBody>
+                <h3>Things to consider when using Ethereum</h3>
+                <ul>
+                  <li>
+                    Each Ethereum transaction requires a fee in the form of ETH,
+                    even if you need to move different tokens built on Ethereum
+                    like the stablecoins USDC or DAI.
+                  </li>
+                  <li>
+                    Fees can be high depending on the demand so we recommend
+                    using <Link to="/layer-2/">Layer 2s</Link>.
+                  </li>
+                  <li>
+                    There are options for custodial and non-custodial wallets.
+                  </li>
+                </ul>
+              </BannerBody>
+              <BannerImage>
+                <GatsbyImage
+                  image={getImage(data.newRings)}
+                  alt=""
+                />
+              </BannerImage>
+            </Banner>
+
+            <AdditionalReadingHeader>
+              Additional reading on Ethereum
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/community/support/">
+                Ethereum and wallet support
```

Comment:
Confusing wording:

is it Ethereum AND wallet support (i.e. Ethereum support and wallet support) or Ethereum and wallet support.

```suggestion
                Support for Ethereum and wallets
```

Relevant meme 😄:

![image](https://user-images.githubusercontent.com/62268199/177806798-44c471c3-6408-4d3a-9db8-2c776b9c5ce2.png)